pandas/_libs/algos_rank_helper.pxi.in

```diff
@@ -122,38 +116,30 @@ def rank_1d_{{dtype}}({{ctype}}[:] in_arr, ties_method='average',
     non_na_idx = _indices[0] if len(_indices) > 0 else -1
     argsorted = _as.astype('i8')
 
-    {{if dtype == 'object'}}
-    if True:
-    {{else}}
-    with nogil:
-    {{endif}}
-        # TODO: why does the 2d version not have a nogil block?
+    if rank_t is object:
+        # TODO: de-duplicate once cython supports conditional nogil
         for i in range(n):
             sum_ranks += i + 1
             dups += 1
 
-            {{if dtype == 'object'}}
-            val = util.get_value_at(sorted_data, i)
```

Comment:
Is this line no longer required?

Reply:
AFAICT it hasnt been needed for a while.  I recently removed most usages of util.get_value_at as unnecessary when we know `i` is an integer.